sql/src/test/java/io/crate/integrationtests/SQLTransportIntegrationTest.java

```diff
@@ -549,4 +555,34 @@ protected String getIndexSettings(String index) throws IOException {
         return sqlOperations.createSession(defaultSchema, null, options, DEFAULT_SOFT_LIMIT);
     }
 
+    /**
+     * @return true if a class or method in the stacktrace contains a @UseJdbc annotation
```

Comment:
drop the stracktrace documentation as we're not using that anymore (also, please rewrite the docs as they don't make sense as they are now eg "and based on the ration provided" - what happens? it doesn't tie into the rest)